test/testlib/nuodb_database_utilities.go

```diff
@@ -355,3 +357,54 @@ func ServePodFileViaHTTP(t *testing.T, namespaceName string, srcPodName string,
 	k8s.RunKubectl(t, kubectlOptions, "cp", localFilePath, nginxPod+":/usr/share/nginx/html/"+fileName)
 	return fmt.Sprintf("http://nginx.%s.svc.cluster.local/%s", namespaceName, fileName)
 }
+
+func GetNuoDBVersion(t *testing.T, options *helm.Options) string {
+	kubectlOptions := k8s.NewKubectlOptions("", "", "default")
```

Comment:
the idea behind randomly generated namespaces was to make sure that multiple tests can be run at the same time. We don't do that much, since we dont have large available multi-tenant clusters but I would like to keep that option open. If multiple tests execute in the "default" namespace, they might collide and fail randomly.

Reply:
good point. I will generate random namespace name for this.